swagger/v1/swagger.yaml

```diff
@@ -709,6 +709,10 @@ servers:
   variables:
     localHost2:
       default: 127.0.0.1:3000/
+- url: http://{Lessoner Prod Render}
+  variables:
+    rend:
+      default: lessoner-project-backend.onrender.com/
```

Comment:
I think we need to add it to `spec/swagger_helper.rb`, then generate `swagger.yaml`.